main.py

```diff
@@ -24,17 +25,130 @@ def setup(self):
+                command = flow.get('command')
+                flows_dict = flow.get('data')
+                self._install_flows(command, flows_dict, [switch])
+            self.resent_flows.add(dpid)
+            log.info(f'Flows resended to Switch {dpid}')
+        return None
+
+    # pylint: disable=attribute-defined-outside-init
+    def _load_flows(self):
+        """Load stored flows."""
+        try:
+            data = self.storehouse.get_data()['flow_persistence']
+            if 'id' in data:
+                del data['id']
+            self.stored_flows = data
+
+        except KeyError as error:
+            log.info(f'There are no flows to load : {error}')
+        else:
+            log.info('Flows loaded.')
+
+    @staticmethod
+    def _generate_match_fields(flows):
+        """Generate flow match fields."""
+        match_fields = {}
+        for fields in flows.get('flows', {}):
+            if 'priority' in fields:
+                match_fields['priority'] = fields['priority']
+            if 'cookie' in fields:
+                match_fields['cookie'] = fields['cookie']
+            if 'match' in fields:
+                for field, value in fields['match'].items():
+                    match_fields[field] = value
+        return match_fields
+
+    def _store_changed_flows(self, command, flows, switches):
+        """Store changed flows."""
+        store_box_updated = self.stored_flows.copy()
+        # if the flow has a destination dpid it can be stored.
+        if not switches:
+            log.info('The Flow cannot be stored, the destination Switches '
+                     f'have not been specified: {switches}')
+            return None
+        for switch in switches:
+            new_flow = {}
+            flow_list = []
+            new_flow['command'] = command
+            # The fields to check if the flow is already stored.
+            new_flow['match_fields'] = self._generate_match_fields(flows)
+            new_flow['data'] = flows
+
+            if switch.id not in store_box_updated:
+                # Switch not stored, add to box.
+                flow_list.append(new_flow)
+                store_box_updated[switch.id] = {"flow_list": flow_list}
+                continue
+
+            stored_flows = store_box_updated[switch.id].get('flow_list', [])
+
+            # Check if flow already stored
+            for stored_flow in stored_flows:
+
+                new_flow_match_fields = new_flow.get('match_fields')
+                stored_flow_match_fields = stored_flow.get('match_fields')
+
+                if new_flow_match_fields == stored_flow_match_fields:
+
+                    new_flow_command = new_flow.get('command')
+                    stored_flow_command = stored_flow.get('command')
+
+                    if new_flow_command == stored_flow_command:
+                        log.debug('Data already stored.')
+                        return None
+                    else:
```

Comment:
There's no need for an `else` after a return.

Reply:
Fixed in the last commit.